style.css

```diff
@@ -0,0 +1,214 @@
+
+body{
+    background: #EEE;
+}
+#header {
+    display: flex;
+    justify-content: space-between;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    right: 0;
+    left: 0;
+    height: auto;
+    z-index: 1;
+    background-color: #EEE;
+    
+    
+}
+#header-img {
+   width : 300px;
+   font-family: 'Lato', sans-serif;
+   padding: 20px;
+   
+}
+
+header nav{
+    display: flex;
+    line-height: 60px;
+    padding-right: 30px;
+}
+header nav a{
+    text-decoration: none;
+    color: black;
+    padding: 7px 13px;
+    margin: 0 50px;
+
+}
+
+
+main{
+
+    text-align: center;
+    display: flex;
+    flex-direction: column;
```

Comment:
- [ OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
  justify-content: center;
  flex-direction: column;
}
```
- HTML file
```
<div class="d-flex">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉